module/VuFind/src/VuFind/I18n/Translator/LanguageInitializerTrait.php

```diff
@@ -51,7 +51,7 @@ protected function getTextDomains()
         $base = APPLICATION_PATH;
         $local = LOCAL_OVERRIDE_DIR;
         $languagePathParts = ["$base/languages"];
-        if (!empty($local)) {
+        if (strlen($local) > 0) {
```

Comment:
What is the motivation for this change? We use `empty` to check for empty strings in quite a few places, and it seems like a reasonable approach as long as there is no realistic chance of other false-y values being included in the string. Is there a particular reason why phpstan doesn't like it here but isn't complaining about other instances?

Reply:
I couldn't find the real problem behind this. But I think, this is problem only for phpstan, as it looks on `$local` as it would be constant and thinks it is always not empty, but I am not sure why. Other question is, if we need the the check of emptiness at all as this is the only place in code, we are checking the LOCAL_OVERRIDE_DIR constant to not be empty...